server/src/main/scala/za/co/absa/atum/server/api/repository/PartitioningRepositoryImpl.scala

```diff
@@ -195,7 +226,10 @@ object PartitioningRepositoryImpl {
       getPartitioningMeasuresById,
       getPartitioning,
       getFlowPartitionings,
+      getAncestors,
+      //patchPartitioningParent,
```

Comment:
these patch ones are not relevant in this PR here, I'd like to merge only live code into master unless there is a good reason for it, ideally properly documented. But here I think it's better to remove these